CHANGELOG.yaml

```diff
@@ -4,7 +4,7 @@ meta:
   issues: a list of ticket URLs (GitHub, JIRA, ...) related to the change
   upgrade: a list of actions required to upgrade to a version that includes this change
   It may be any combination of the following values:
-    - deploy: (requires `make deploy`)
+    - package: (requires `make package`)
```

Comment:
Except for this instance, I'm not sure it makes sense to retroactively change this CHANGELOG entries. Note that I'm not requesting you to change this, but I'm just expressing my doubt.